languages.toml

```diff
@@ -2099,6 +2099,24 @@ roots = []
 name = "xml"
 source = { git = "https://github.com/RenjiSann/tree-sitter-xml", rev = "48a7c2b6fb9d515577e115e6788937e837815651" }
 
+
+[[language]]
+name = "dtd"
+scope = "source.dtd"
+injection-regex = "dtd"
+file-types = ["dtd", "ent"]
+indent = {tab-width = 2, unit = "  "}
+roots = []
+
+[language.auto-pairs]
```

Comment:
Why remove the auto-pairs for `'` and `"`?

Reply:
Excellent question! I don't know where those went, I'll add them back in :)